test/unit/user_event/click_test.dart

```diff
@@ -97,6 +97,27 @@ void main() {
         _verifyClickEvent(clickCount: clickCount);
       });
 
+      test('skipPointerEventsCheck', () {
+        var wasClicked = false;
+
+        // Render into a local view because there isn't a great
+        // way to test pointer events with shared test logic
+        final localView = rtl.render(react.button({
+          'data-test-id': 'the-local-button',
```

Comment:
#nit you could probably avoid having to make a separate view and getting by test-id if you use `view.rerender` instead.